build/assets/css/styles.css

```diff
@@ -12,20 +12,45 @@ body {
 }
 .markdown-section ol, .markdown-section p, .markdown-section ul {
   line-height: 1.5;
+  word-spacing: 0;
 }
 .markdown-section blockquote {
   border-left-color: #858585;
+  color: #243746;
+}
+.markdown-section blockquote p, .markdown-section em {
+  color: #243746;
 }
 .sidebar, .sidebar input {
-  background: #243746;
-  color: #f9f8f5;
+  background: #09615D;
+  color: #FFB259;
 }
 .sidebar input::placeholder {
   color: #f9f8f5c4;
 }
+.sidebar ul li {
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+.sidebar ul li a, .app-nav ul li a {
+  font-family: Poppins, sans-serif;
+  text-decoration: none;
+  border-bottom: 2px solid transparent;
+  display: inline;
+}
 .sidebar ul li a {
-  color: #f9f8f5;
+  color: #fff;
```

Comment:
Raw white? Is that intentional?

Reply:
Good spot! I changed it after I was colour contrast issues and forgot to put it back.

Reply:
Gaz is happy with this being AA, so will change this back to `#f9f8f5`.